internal/dao/helm.go

```diff
@@ -31,7 +31,18 @@ func (h *Helm) List(ctx context.Context, ns string) ([]runtime.Object, error) {
 	if err != nil {
 		return nil, err
 	}
-	rr, err := action.NewList(cfg).Run()
+
+	// List all helm releases
+	client := action.NewList(cfg)
```

Comment:
Hi @derailed 
> First off I think client.All = true would do what you are trying to achieve

I agree

> I can already ear folks wanting to filter selectively as this could show much more info than they would care to see

I agree that people will want to filter, but I think showing everything is better than being selective until the filter exists. This only adds `pending-` and `failed` charts which people likely want to be aware of.

> best to pass-in the viewing options at the api level

I am not sure how to go about this. Golang is not my strength. Can you please point me in the right direction or feel free to take over?